frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -208,5 +211,8 @@ async function checkCellTypes(page: Page, cellTypes = CELL_TYPE_FILTERS) {
  * Remove cell filter
  */
 async function removeCellFilter(page: Page) {
-  await page.getByTestId("CancelIcon").nth(0).click();
+  await await page
```

Comment:
maybe remove extra `await` 😄 ?

Reply:
👍🏽

Reply:
that's funny that the linter didn't flag that - fixed!